components/builder-sessionsrv/src/migrations/sessions.rs

```diff
@@ -120,5 +123,47 @@ pub fn migrate(migrator: &mut Migrator) -> SrvResult<()> {
                  $$"#)?;
     migrator.migrate("accountsrv",
                  r#"ALTER TABLE IF EXISTS account_sessions DROP CONSTRAINT IF EXISTS account_sessions_account_id_fkey"#)?;
+
+    // Cleanup after table deprecation
+    migrator.migrate(
+        "accountsrv",
+        "DROP TABLE IF EXISTS account_sessions CASCADE",
+    )?;
+    migrator.migrate(
+        "accountsrv",
+        r#"DROP FUNCTION IF EXISTS insert_account_session_v2(
+                    a_account_id bigint,
```

Comment:
I dunno if you care, but technically you only need the type here, not the name.  That applies to all these DROP statements.